src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ModelBinderProviderCollection.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Represents a collection of model binder providers.
+    /// </summary>
+    public class ModelBinderProviderCollection : Collection<IModelBinderProvider>
```

Comment:
This type lives in `Microsoft.AspNetCore.Mvc.Core`. Is that correct? I noticed that some collection types are in `Abstractions` and some in `Core`.